components/Home/HomeDownloadButton.tsx

```diff
@@ -32,23 +33,23 @@ const HomeDownloadButton: FC<NodeRelease> = ({
         title={nodeDownloadTitle}
         data-version={versionWithPrefix}
       >
-        {version} {labels[isLts ? 'lts' : 'current']}
-        <small>{labels[`tagline-${isLts ? 'lts' : 'current'}`]}</small>
+        {version} {labels?.[isLts ? 'lts' : 'current']}
```

Comment:
so many optional chains in this file make me wonder if it should be short circuited once at the top level?